### PR TITLE
phpunit6 support

### PR DESCRIPTION
**Pull request by omissis**
This PR introduces phpunit6 support and adds PSR-2 compliance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,19 +6,13 @@ Inspired by [Kris Wallsmith faster PHPUnit article](http://kriswallsmith.net/pos
...
+Install it via composer:
+
+    composer require "padraic/phpunit-accelerator:^2.0"
 
-Update the vendor libraries
+Then update the vendor libraries
 
-    curl -s http://getcomposer.org/installer | php
-    php composer.phar install
+    composer install
```
Comment (padraic) [inline README.md]: `composer update`?

**Review by padraic — APPROVED** [review]
Thanks @omissis, I'll merge and tag as a 2.0 when I can.

File: README.md (hunk 1 above)
Comment (omissis) [reply README.md]: +1

File: composer.json [hunk 2]
```diff
@@ -17,13 +17,19 @@
...
     ],
     "require": {
-        "php": ">=5.3.3",
-        "phpunit/phpunit": ">=3.7"
+        "php": "^7.0",
+        "phpunit/phpunit": "^6.0"
     },
     "replace": {
-        "mybuilder/phpunit-accelerator": "1.0.x"
+        "mybuilder/phpunit-accelerator": "2.0.x"
```
Comment (omissis) [inline composer.json]: @padraic I am not sure this is really meaningful, as the original accelerator does not have a 2.0 nor support phpunit6. any recommendation on how to proceed here?

Comment (padraic) [reply composer.json]: I'd remove it altogether in that case.

**Comment by omissis** [comment]
@padraic ok, I have implemented the two CR comments, then added some type-hinting and docblocks, and reworked a bit the travis config. looks good to me now :)

**Comment by omissis** [comment]
I think this has become old enough not to be relevant anymore, closing.

**Pull request closed without merging**